tests/data/formats/SPDXJsonExample2.2.json

```diff
@@ -0,0 +1,231 @@
+{
+    "comment": "This is a sample spreadsheet",
+    "name": "Sample_Document-V2.1",
```

Comment:
The file name says 2.2

Reply:
I added the example file from the spec. Since the python-tools don't support spec 2.2 yet, I had to exclude this file from the tests but the file might be a good starting point to really supprt spec 2.2.